Pod/Classes/WPNavigationMediaPickerViewController.m

```diff
@@ -351,7 +351,7 @@ - (void)mediaPickerController:(nonnull WPMediaPickerViewController *)picker sele
     [self updateSelectionAction];
 }
 
-- (BOOL)mediaPickerController:(nonnull WPMediaPickerViewController *)picker handleError:(nonnull NSError *)error {
+- (BOOL)mediaPickerController:(nonnull WPMediaPickerViewController *)picker handleError:(nullable NSError *)error {
```

Comment:
It's weird that we're calling `mediaPickedController:handleError:` with a `nil` error.  Is there a valid use-case for allowing this scenario?

IMHO it would be far cleaner if we used this logic instead:

1. If there's no error: avoid this whole chain of methods from being called.
2. If there's an error: make sure `error` is properly instantiated.
3. Or if the first two are too tricky to implement, at least make sure we bail out here if error is `nil` without calling the method in the delegate.

Just my 2 cents.

Reply:
Following up from our Slack discussion, good point that it doesn't necessarily make sense to call `handleError` with no error. I'll push a PR shortly with a different approach to fix this crash.